config/base/env/config

```diff
@@ -10,7 +10,8 @@ REST_PORT=8080
 PROMETHEUS_PORT=9090
 TLS_HOSTNAME_OVERRIDE=
 TLS_PATH=/etc/tls
-NO_AUTH=false
+AUTH_DISABLE=false
+AUTH_IMPERSONATE=false
```

Comment:
Personally, I think we should set the default config to `true` in our release. At minimum this will ensure that we run end to end tests with this enabled, reducing our test matrix.

Reply:
Default set to `true`.